### ISSUE TITLE
Inadequate color contrast: search bar and button

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/freedom#38]
The full text site search bar's border and placeholder text have inadequate contrast (#CCCCCC on a #FFFFFF ground). The border should be boosted to at least 3:1 and the placeholder text to at least 4.5:1 (as it appears the text size is under 18pt), though technically so long as the placeholder text gains proper visibility, the border can remain at its current contrast as the text (at a corrected contrast) is sufficient to indicate hit area (for more, see [what WAI has to say on "Boundaries"](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#intent)).

The contrast of the search button is 2.8:1 (at #999999 on #FFFFFF) and should be increased to at least meet the 3:1 guideline. Furthermore, the button undergoes a state change upon hovering, which causes the search icon to fade entirely, transforming into a new state at a 1:1 contrast. If a visual state change is desired here, it must occur while maintaining at least a 3:1 contrast. It is also just a good practice across the board to ensure that color is not used as the only means of communicating, especially in the case of something like a change in state. Some possible alternatives in this case are altering the border or font weight of the icon in the the button upon hover.

**Comment by aviannamiller** [comment]
Excellent, thank you! That's great.
